api/metadata/player_loadout.proto

```diff
@@ -15,9 +15,10 @@ message PlayerLoadout {
     optional int32 topper = 10;
     optional int32 antenna = 11;
     optional int32 engine_audio = 12;
+    optional int32 avatar_border = 22;
 
 
-    optional int32 banner_paint = 13;
+    optional int64 banner_paint = 13;
```

Comment:
Do we know the implications of changing this? Any reason it needs to be 64?

Reply:
If the version with 32 bits was deployed, this change would make it **not** backwards compatible with older files. Banner and avatar border uses ARGB integer in a 0xAARRGGBB format that does not fit in 32 bits.

But I just realized that banner can either have the painted attribute or a custom color selected for them depending on what banner it is so I'll just make a separate field for it.

Also need to check if it's the same with avatar borders.